JS-variables-and-numbers/js-variables-and-numbers_social-media-post/js/index.js

```diff
@@ -18,5 +23,11 @@ console.clear();
 */
 
 // --v-- write your code here --v--
+console.log(title);
+console.log(text);
+console.log(numberOfLikes);
+console.log(author);
+console.log(isReported);
```

Comment:
Hint, you can console.log more than one variable at once by separating them with a comma. console.log(title, text, author, etc) :)
Good job.